HelloWorldFunction/src/main/java/helloworld/App.java

```diff
@@ -26,16 +28,11 @@ public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEv
 
         APIGatewayProxyResponseEvent response = new APIGatewayProxyResponseEvent()
                 .withHeaders(headers);
+        PortfolioRequester generator = new PortfolioRequester();
```

Comment:
1. Rename `generator` to `portfolioProvider`.

Reply:
Solved in the next commit